docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,374 @@
+algorithm B.A.T.M.A.N. V (while the old algorithm is called B.A.T.M.A.N. IV) by
+setting *mesh.batman_adv.routing_algo* to ``"BATMAN_V"``. Note that the new routing
+algorithm is not backwards-compatible, so nodes using different algorithms can
+not interoperate.
+
+.. _releases-v2018.1-site-changes:
+
+Site changes
+************
+
+site.mk
+=======
+
+* Due to improved package dependency handling, the packages
+  *gluon-config-mode-core* and *gluon-setup-mode* do not need
+  to be listed explicitly in *site.mk* anymore; they will be
+  pulled in implicitly.
+* As mentioned above, we recommend to include the *ebtables-limit-arp*
+  feature flag.
+* We recommend to use *GLUON_FEATURES* for all Gluon packages, and rely on
+  *GLUON_SITE_PACKAGES* for non-Gluon (OpenWrt) package only, as explained
+  in :ref:`user-site-feature-flags`.
+
+site.conf
+=========
+
+When updating a site configuration from Gluon 2017.1.x, the following changes
+must be made:
+
+* .. code-block:: lua
+
+    domain_seed = 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+
+  These 32 bytes of random data (encoded in hexadecimal) are used to seed a number
+  of site/domain specific random values that must be the same on all nodes of the
+  same mesh, but different for different meshes. The following command
+  can be used to generate such a random value:
+
+  .. code-block:: shell
+
+    echo $(hexdump -v -n 32 -e '1/1 "%02x"' </dev/urandom)
+
+  In multidomain setups, repeat this command for each domain.
+
+  At the moment, only the VXLAN ID for wired meshing is derived from the domain seed.
+
+* .. code-block:: lua
+
+    mesh = {
+      vxlan = true, -- or false
+      -- ...
+    },
+
+  In single domain setups, the *mesh.vxlan* option is mandatory now. It should be set to *true* in new
+  meshes. Existing setups should set it to *false* to retain compatibility with older versions of Gluon.
+
+  In multidomain setups, *mesh.vxlan* defaults to *true* and does not need to be set explicitly.
+  It can still be set to *false* for individual domains that should allow wired meshing with existing
+  setups.
+
+* Password change form
+
+  The password change from in the "Advanced settings" is not shown by default anymore, as SSH keys are
+  the recommended means of authentication. It is still possible to set a password via SSH while in
+  config mode.
+
+  Set
+
+  .. code-block:: lua
+
+    config_mode = {
+      remote_login = {
+        show_password_form = true,
+        -- ...
+      },
+      -- ...
+    },
+
+  to restore the old behaviour.
+
+  When shown, the password form requires a minimum password length of 12 characters now. This requirement
+  can be modified using the *config_omde.remote_login.min_password_length* setting.
+
+i18n
+====
+
+It is now possible to override a few labels and desciptions in the configuration
+wizard. The available message IDs are listed in :ref:`site-config-mode-texts`.
```

Comment:
Message-IDs
https://en.wikipedia.org/wiki/Message-ID

Reply:
Gettext message IDs don't have anything to do with email/... message IDs. The gettext docs use "msgids", but we already have "message IDs" in the referenced doc pages, so I'd leave it as it is for now.